pcdsdevices/epics_motor.py

```diff
@@ -221,6 +186,85 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_limit(self):
+        """Get the low limit."""
+        return self.low_limit_travel.get()
+
+    def set_low_limit(self, value):
+        """
+        Set the low limit.
+
+        Parameters
+        ----------
+        value : float
+            Limit of travel in the negative direction.
+
+        Raises
+        ------
+        ValueError
+            When motor in motion or position outside of limit.
+        """
+        if not self.moving:
+            # update EPICS
+            _current_high_limit = self._get_epics_limits()[1]
+            if value <= self.position:
+                if value < _current_high_limit:
+                    self.low_limit_travel.put(value)
+                else:
+                    raise ValueError('Could not set motor low limit to %f. '
```

Comment:
This might be a bit too much, I probably could have combined the if statements here but I though raising more specific errors is not a bad idea?

Reply:
I missed this when looking earlier. Descriptive errors are very helpful 👍